doc/source/getting_started/intro_tutorials/04_plotting.rst

```diff
@@ -17,26 +17,8 @@
         </div>
         <ul class="list-group list-group-flush">
             <li class="list-group-item">
-                <div data-toggle="collapse" href="#collapsedata" role="button" aria-expanded="false" aria-controls="collapsedata">
-                    <span class="badge badge-dark">Air quality data</span>
-                </div>
-                <div class="collapse" id="collapsedata">
-                    <div class="card-body">
-                        <p class="card-text">
 
-For this tutorial, air quality data about :math:`NO_2` is used, made
-available by `openaq <https://openaq.org>`__ and using the
-`py-openaq <http://dhhagan.github.io/py-openaq/index.html>`__ package.
-The ``air_quality_no2.csv`` data set provides :math:`NO_2` values for
-the measurement stations *FR04014*, *BETR801* and *London Westminster*
-in respectively Paris, Antwerp and London.
-
-.. raw:: html
-
-                        </p>
-                    <a href="https://github.com/pandas-dev/pandas/tree/master/doc/data/air_quality_no2.csv" class="btn btn-dark btn-sm">To raw data</a>
-                </div>
-            </div>
+. include:: air_quality_no2.rst
```

Comment:
There is a missing second dot here, I think (at the beginning of the line)

Reply:
I need to stop making pull requests when sleep deprived 🤦 Fixed in https://github.com/pandas-dev/pandas/pull/38839.